### PR TITLE
Add travel and accommodation info

### PR DESCRIPTION
**Pull request by Lucie-H**

### CODE REVIEW

File: info.html [hunk 1]
```diff
@@ -91,6 +92,62 @@ <h2 class="color--blue">
...
+        <div class="col-xs-12 col-sm-10 col-md-8 col-md-offset-1">
+          <h2 class="h1 color--blue">
+            Travel &amp; Transport
+          </h2>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-xs-12 col-sm-8 col-sm-offset-2">
+          <h2 class="color--blue">Berlin’s Airports</h2>
+          <p>Berlin has two airports, both are located close to the city center, a 30-minute taxi ride from the venue (ca. EUR 30). There are also trains and buses that connect the airports to Berlin’s city center.</p>
```
Comment (sfrdmn) [inline info.html]: both of which

File: info.html [hunk 2]
```diff
@@ -91,6 +92,62 @@ <h2 class="color--blue">
...
+          <ul class="list-simple">
+            <li><a href="http://www.berlin-airport.de/en/travellers-txl/to-and-from/buses-and-trains/index.php" target="_blank">Berlin-Tegel<small> (TXL)</small></a></li>
+            <li><a href="http://www.berlin-airport.de/en/travellers-sxf/to-and-from/buses-and-trains/index.php" target="_blank">Berlin-Schönefeld<small> (SXF)</small></a></li>
+          </ul>
+          <h2 class="color--blue">Public Transport</h2>
+          <p>
+            Berlin has many public transport options: Subway, trains, buses, trams are all operated by <a href="http://www.bvg.de/en" target="_blank">BVG</a>. We recommend getting one of the <a href="https://shop.bvg.de/index.php/group/73" target="_blank">tourist tickets</a> for the length of your stay – this will allow you to use all subway trains, trams and buses in all of Berlin, plus it comes with discounts for popular sights, museums, and other attractions.
```
Comment (sfrdmn) [inline info.html]: the subway, trains, buses, and trams

File: info.html [hunk 3]
```diff
@@ -91,6 +92,62 @@ <h2 class="color--blue">
...
+      <div class="row">
+        <div class="col-xs-12 col-sm-10 col-md-8 col-md-offset-1">
+          <h2 class="h1 color--blue">
+            Accommodation
+          </h2>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-xs-12 col-sm-8 col-sm-offset-2">
+          <p>Below are a two hotels we recommend, both a little walk away from our venue. If none of those fit your needs, check out <a href="http://www.hrs.com/" target="_blank">HRS</a> or private accommodation options like <a href="https://www.airbnb.com/locations/berlin" target="_blank">AirBnB</a> or <a href="https://www.couchsurfing.org/" target="_blank">Couchsurfing</a>.</p>
```
Comment (sfrdmn) [inline info.html]: short walk

Comment (sfrdmn) [reply info.html]: neither of those

File: info.html [hunk 4]
```diff
@@ -91,6 +92,62 @@ <h2 class="color--blue">
...
+          <ul class="list-simple">
+            <li><a href="https://www.nhow-berlin.com/en/" target="_blank">Hotel NHow <span>(20 min walk)</span></a></li>
+            <li><a href="http://michelbergerhotel.com/en//" target="_blank">Michelberger Hotel <span>(20 min walk)</span></a></li>
```
Comment (lukaszklis) [inline info.html]:
```diff
- //
+ /
```

in the end of the `href` attribute

File: info.html [hunk 5]
```diff
@@ -91,6 +92,62 @@ <h2 class="color--blue">
...
+          </ul>
+          <p>
+            If you’d like to team up with other attendees to travel to or stay in Berlin, let us and the community know on twitter: <span class="color--green">#cssconfeu</span> – we will help spread the word!
```
Comment (sfrdmn) [inline info.html]: capitalize Twitter?

Comment (lukaszklis) [inline info.html]:
```diff
- twitter
+ Twitter
```

**Review by sfrdmn — APPROVED** [review]
Looks good! Just reviewed the content though

**Review by lukaszklis — APPROVED** [review]

File: info.html (hunk 2 above)
Comment (sfrdmn) [reply info.html]: or maybe actually "many transport options _punkt_ The subway, buses, etc"

**Comment by Lucie-H** [comment]
@sfrdmn @lukaszklis Thank you! Updated it.

**Comment by lukaszklis** [comment]
Looks good! :+1:

**Comment by Lucie-H** [comment]
@lukaszklis who's merging, do we have a workflow there? :)

**Comment by lukaszklis** [comment]
Just do it. :) Whenever at least one team member looked at a PR, it's good
to go.

**Pull request merged**